exporting/pubsub/README.md

```diff
@@ -0,0 +1,34 @@
+<!--
+---
+title: "Export metrics to Google Cloud Pub/Sub Service"
+custom_edit_url: https://github.com/netdata/netdata/edit/master/exporting/pubsub/README.md
+---
+-->
+
+# Export metrics to Google Cloud Pub/Sub Service
+
+## Prerequisites
+
+To use the Pub/Sub service for metric collecting and processing, you should first
+[install](https://github.com/googleapis/cpp-cmakefiles) Google Cloud Platform C++ Proto Libraries.
+Pub/Sub support is also dependent on the dependencies of those libraries, like `protobuf` and `grpc`. Next, Netdata
+should be re-installed from the source. The installer will detect that the required libraries are now available.
+
+## Configuration
+
+To enable data sending to the Pub/Sub service, run `./edit-config exporting.conf` in the Netdata configuration directory
+and set the following options:
+
+```conf
+[pubsub:my_instance]
+    enabled = yes
+    destination = pubsub.googleapis.com
+    credentials file = /etc/netdata/google_cloud_credentials.json
+    project id = my_project
+    topic id = my_topic
+```
+
+Set the `destination` option to a Pub/Sub service endpoint. `pubsub.googleapis.com` is the default one.
+You should provide access to the file with Google Cloud credentials for the `netdata` user.
```

Comment:
We can start with something like this. I may not have the permissions right—feel free to correct me if I'm wrong!

```suggestion
Next, create the credentials JSON file by following Google Cloud's [authentication guide](https://cloud.google.com/docs/authentication/getting-started#creating_a_service_account). The user running the Agent (typically `netdata`) needs read access to `google_cloud_credentials.json`, which you can set with `chmod 644 google_cloud_credentials.json`.
```

Reply:
Done.